.github/workflows/build_and_push_ci_image.yml

```diff
@@ -138,6 +138,44 @@ jobs:
         tags: occlumbackup/occlum:${{ github.event.inputs.tag }}-ubuntu18.04-openvino
 
 
+  Build_openvino2021_image:
```

Comment:
You should not add anything in this yaml file. It is not part of the auto-test flow.